docs/building-your-quest/triggers-and-payloads.md

```diff
@@ -118,6 +118,21 @@ trigger:
         - actionId: a different action here
 ```
 
+### USER_EVENT
+
+Triggered when there's a custom event called on the user, you can trigger it by calling `https://engine.wilco.gg/users/${WILCO_USER_ID}/event`. You can find the user id in the `.wilco` file in the user's repository.
+This trigger must contain `params` with a `event` key that matches the event prop in the payload of the event that you're sending.
+For example:
+
+```yaml
+trigger: 
+  type: user_event
```

Comment:
Not a fan of this name :(
Every trigger is basically a user event.

Reply:
have other suggestion?